Makefile

```diff
@@ -97,7 +97,7 @@ test-ci: test/markdown test/sanity test/unit install test/subcommand test/e2e
 
 test/ci-go: test/subcommand test/e2e/go
 
-test/ci-ansible: test/e2e/ansible test/e2e/ansible-molecule
+test/ci-ansible: test/e2e/ansible-molecule
```

Comment:
After this, will anything use `test/e2e/ansible` anymore? I guess developers might still run it manually? Our Makefile has been growing with the new Openshift CI tasks, and I just want to make sure everything in here still has a purpose.

Also, does `doc/dev/testing/running-the-tests.md` need to be updated?

Reply:
`test/e2e/ansible` isn't used anymore if we do this. Developers could run it manually though. The main question now would be should we have devs run the tests in a similar way to how Openshift-CI does (build all the images first, and then run the test; we might need to make some changes to make it compatible with the old `minikube docker-env` testing, which currently doesn't work because out pull policy is `Always`). If we decide that devs should use the "build all the images first and then test" method, we can remove the old ansible test scripts and another make target or 2.

`doc/dev/running-the-tests.md` should be updated. We'll probably have to add new documentation for running the new tests locally. But like I said, the tests are not designed in a way to run correctly locally without modification, so that's not really something we can do at the moment.

I'll update the documentation that changes with this PR, and I'll make another PR to allow devs to run the new tests, along with documentation for that.